dandi/model_types.py

```diff
@@ -46,20 +46,32 @@
         {
             "@id": "dandi:sha1",
             "@type": "dandi:DigestType",
-            "rdfs:comment": "SHA-1 checksum",
-            "rdfs:label": "SHA-1",
+            "rdfs:comment": "SHA1 checksum",
+            "rdfs:label": "SHA1",
         },
         {
-            "@id": "dandi:sha256",
+            "@id": "dandi:sha2-256",
             "@type": "dandi:DigestType",
-            "rdfs:comment": "SHA-256 checksum",
-            "rdfs:label": "SHA-256",
+            "rdfs:comment": "SHA2-256 checksum",
+            "rdfs:label": "SHA2-256",
         },
         {
-            "@id": "dandi:sha512",
+            "@id": "dandi:sha3-256",
             "@type": "dandi:DigestType",
-            "rdfs:comment": "SHA-512 checksum",
-            "rdfs:label": "SHA-512",
+            "rdfs:comment": "SHA3-256 checksum",
+            "rdfs:label": "SHA3-256",
+        },
+        {
+            "@id": "dandi:blake2b-256",
+            "@type": "dandi:DigestType",
+            "rdfs:comment": "BLAKE2B-256 checksum",
+            "rdfs:label": "BLAKE2B-256",
+        },
+        {
+            "@id": "dandi:blake3-256",
```

Comment:
```suggestion
            "@id": "dandi:blake3",
```
if we are to align to that https://github.com/multiformats/multicodec/blob/master/table.csv#L21 where they use just `blake3` for the one with 32 bytes (==256bits).  And according to https://en.wikipedia.org/wiki/BLAKE_(hash_function)#BLAKE3  I do not see any other size possible.  
Probably should be reflected also in the rdfs:label as well (i.e. drop -256)